Library/Homebrew/utils/diskusage.rb

```diff
@@ -0,0 +1,57 @@
+require "find"
+
+module Utils
+  class DiskUsage
```

Comment:
I don't think I follow your thought of replacing `Pathname.abv`, please explain. 

As I see, `Pathname.abv` is just a display method but we need `Fixnum` (disk usage) values for all the `path`s which are to be cleared, so that, a total can be calculated without parsing output of `Pathname.abv`. I'll check, and try to modify `extend/pathname.rb` to accommodate this shortcoming.